libs/azure-testing/src/testFixtures/java/io/crate/azure/testing/AzureHttpHandler.java

```diff
@@ -53,10 +53,16 @@ public class AzureHttpHandler implements HttpHandler {
 
     private final Map<String, BytesReference> blobs;
     private final String container;
+    private final boolean addDateHeader;
 
-    public AzureHttpHandler(final String container) {
+    /**
+     * @param addDateHeader indicates whether to add Last-Modified header to the list API response.
+     * COPY/OpenDAL requires 'Last-Modified' in the list API response.
+     */
+    public AzureHttpHandler(final String container, boolean addDateHeader) {
```

Comment:
>Isn't it possible to always set this header?

No, AzureSnapshotIntegrationTest fails otherwise. Reason is that different client is used there and apparently clients are header-sensitive